Source/Blazorise/Components/Table/TableRowGroup.razor

```diff
@@ -19,6 +19,15 @@
     @oncontextmenu="@OnContextMenuHandler"
     @oncontextmenu:preventDefault="@ContextMenuPreventDefault"
     @attributes="@Attributes">
+    @for ( int i = 0; i < IndentTableCells; i++ )
+    {
+        <td>
```

Comment:
Add empty cell class name here?

Reply:
Suggestions for naming? just empty? b-cell-empty?

Reply:
![image](https://user-images.githubusercontent.com/900302/212478996-ff5896bc-e953-4cd4-95e6-583a8cc0cb54.png)

Add a third class name and inherit for each provider. Can be named `table-group-indentcell`.